custom_components/frigate/media_source.py

```diff
@@ -461,13 +463,27 @@ def get_integration_proxy_path(self) -> str:
             and self.hour is not None
         ):
             year, month, day = self.year_month_day.split("-")
+            tz_offset = int(
+                (
+                    dt.datetime.now(
+                        pytz.timezone(timezone.replace(",", "/"))
+                    ).utcoffset()
+                    or dt.timedelta()
+                ).total_seconds()
+                / 60
+                / 60
+            )
+            start_date = dt.datetime(
+                int(year), int(month), int(day), int(self.hour), tzinfo=dt.timezone.utc
+            ) - dt.timedelta(hours=tz_offset)
```

Comment:
Can we just specify the timedelta in seconds, and avoid the double divisions above?

Reply:
Good point